.wiki/Configuration.md

```diff
@@ -0,0 +1,35 @@
+The main Artipie configuration files is `/etc/artipie/artipie.yml`. It contains server meta configuration, such as:
+ - `layout` - `flat` or `org` string
+ - `storage` - repositories definition storage config
+ - `credentials` - user credentials config
+
+Example: 
+```yaml
+# /etc/artipie/artipie.yml
+meta:
+  layout: org
+  storage:
+    type: fs
+    path: /tmp/artipie/data/my-docker
+  credentials:
+    type: file
+    path: _credentials.yml
+```
+
+Layout specifies the URI path layout for Artipie. In case of `flat`, Artipie provides repositories at first path level, e.g. `artipie.host/repo1`, `artipie.host/repo2`. Only Artipie administrator can manage repositories in flat layout, users can't create new repositories or delete by self. `org` layout provides hierarchy support for repositories: URI path has two parts for org layouts: `<org>/<repo>`, where `<org>` is oganisation name, and `<repo>` the name of repository, e.g. `artipie.host/artipie/maven` - `maven` repository of `artipie` organisation. In org layout, organisation may have a maintainer who can manage repositories and permissions within organisation, it can add, delete and edit repositories, add granular permissions for users for each repository.
+
+Storage - is a [storage configuration](https://github.com/artipie/artipie/wiki/Configuration-Storage)
+for [repository definitions](https://github.com/artipie/artipie/wiki/Configuration-Repository).
+It locates a storage where all config files for each repository are located. Keep in mind,
+Artipie user should have read and write permissions for this storage.
+
+Credentials - provider for user credentials, it can be managed only by Artipie server administrator.  The type of credentials config could be `file` (support other types such as database or LDAP will be added later). For file credentials, it has a `path` parameter - it's a relative key for repository storage configuration. It contains passwords validators for username keys in this format:
+```yaml
+credentials:
+  user1:
+    pass: "sha256:abc...123"
+  user2:
+    pass: "plain:secret-password"
+```
```

Comment:
@g4s8 we support more readable format for `credentials` now, example can be found [here](https://github.com/artipie/artipie#multitenancy)